README.md

```diff
@@ -367,7 +393,7 @@ data.
 > [official GitHub documentation](https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)
 > to get started setting up your own self-hosted runner.
 
-### Allocating cloud resources with CML
+#### Allocating Cloud Compute Resources with CML
```

Comment:
Why Compute?

Reply:
Because that's what we do?

Reply:
Im OK but sounds a bit redundant to me

Reply:
Agreed! I don't have any strong preference on this, so let's see what @casperdcl thinks.

Reply:
Here is capitalised opposed to pull requests? Why one is capitalised and not the other?

Reply:
I assume that is because headings and titles go in title case. 🤔  Are we following a specific style guide?

Reply:
Titles are capitalised? Not in spanish I guess

Reply:
Exactly! Unlike in Spanish<sup>1</sup>, titles usually are written in [title case](https://en.wikipedia.org/wiki/Title_case), and different style guides recommend different levels of atrociousness.

***

<sup>1</sup> Actually, theay also capitalize language names: it's _Spanish_, not _spanish_.

Reply:
- I was using title case for headings, yes
- there was a lot of confusion in the readme about "cloud" sometimes meaning *storage* and sometimes meaning *compute*. Explicitly nuking that source of ambiguity here.

Reply:
> * _I was using title case for headings, yes_

I'm not exactly a devotee of title case, but, if we're going to use it, it would be nice to apply it consistently to all the headings or, at least, to all the headings with the same level.

> * _there was a lot of confusion in the readme about "cloud" sometimes meaning storage and sometimes meaning compute. Explicitly nuking that source of ambiguity here._

:+1:

Reply:
> would be nice to apply it consistently

er, it is? Apart from `Table of contents` which I'll fix.